instrumentation/cassandra/cassandra-4.4/testing/src/main/java/io/opentelemetry/testing/cassandra/v4_4/AbstractCassandra44Test.java

```diff
@@ -154,10 +151,11 @@ private static Stream<Arguments> provideReactiveParameters() {
                     "users"))));
   }
 
-  @Override
-  protected CqlSessionBuilder addContactPoint(CqlSessionBuilder sessionBuilder) {
```

Comment:
I think we have seen something similar before. I suspect that the issue in in docker/testcontainers network magic where local ip addresses are added/removed by the tests. Probably here it ends up connecting to one of these addresses (or perhaps even an address that just got removed?). As the exception only logs the endpoint we don't see where it actually tried to connect. I'd try
```
    sessionBuilder.addContactEndPoint(new SniEndPoint(address, "localhost") {
      @Override
      public InetSocketAddress resolve() {
        return address;
      }
    });
```